packages/vscode-ui5-language-assistant/README.md

```diff
@@ -47,6 +47,9 @@ The feature is available in the following:
   - Enum values
   - Boolean values
   - Namespaces fully qualified names
+  - metaPath values in macros
```

Comment:
Shouldn't we use `building blocks` instead of `macros`?

Reply:
I checked, we should use Building blocks indeed, thanks